### PR TITLE
cleaned code Solution add fixed bug with EventToGo

### LINKED ISSUE #112: Fix ruleset
- fix ruleset to not violate private fields starting with underscore
- fix ruleset to not violate file header and file summary

### LINKED ISSUE #114: List of "Events" contains invalid data
Preconditions
Sign in to the system

Steps to reproduce
1. Choose the 'Profile' item in the navigation menu
2. Navigate to the 'Events to go' item of the horizontal menu
3. The list of events contains data from the list of "Visited events"
bug_4

### PR DESCRIPTION
**Pull request by ArdeaSt**
Close #112  and Close #114

### CODE REVIEW

File: EventsExpress.Core/IServices/ICountryService.cs [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+using EventsExpress.Db.Entities;
 
 namespace EventsExpress.Core.IServices
 {
     public interface ICountryService
     {
         IEnumerable<Country> GetCountries();
-        //Country GetByName(string name);
+
+        // Country GetByName(string name);
```
Comment (iuriikhrystiuk) [inline EventsExpress.Core/IServices/ICountryService.cs]: Remove commented code, please.

File: EventsExpress.Core/Services/CategoryService.cs [hunk 2]
```diff
@@ -1,92 +1,89 @@
...
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace EventsExpress.Core.Services
 {
     public class CategoryService : ICategoryService
     {
-        private readonly IUnitOfWork Db;
+        private readonly IUnitOfWork db;
```
Comment (iuriikhrystiuk) [inline EventsExpress.Core/Services/CategoryService.cs]: Please start the private field with underscore.

File: EventsExpress.Core/Services/CityService.cs [hunk 3]
```diff
@@ -1,37 +1,35 @@
...
-using System;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace EventsExpress.Core.Services
 {
     public class CityService : ICityService
     {
-        private readonly IUnitOfWork Db;
+        private readonly IUnitOfWork db;
```
Comment (iuriikhrystiuk) [inline EventsExpress.Core/Services/CityService.cs]: Same as above.

File: EventsExpress.Core/Services/CommentService.cs [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
         private readonly IMapper _mapper;
-        
 
         public CommentService(IUnitOfWork uow, IMapper mapper)
         {
             Db = uow;
             _mapper = mapper;
         }
 
+        public IUnitOfWork Db { get; set; }
```
Comment (iuriikhrystiuk) [inline EventsExpress.Core/Services/CommentService.cs:24]: Does it really need to be public? Probably make it a private field.

File: EventsExpress.Core/Services/Custom.ruleset [hunk 5]
```diff
@@ -0,0 +1,3 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (iuriikhrystiuk) [inline EventsExpress.Core/Services/Custom.ruleset]: Remove this file please.

File: EventsExpress.Core/Services/RoleService.cs [hunk 6]
```diff
@@ -1,21 +1,20 @@
...
     public class RoleService : IRoleService
     {
-        public IUnitOfWork Db { get; set; }
-
         public RoleService(IUnitOfWork uow)
         {
             Db = uow;
         }
 
+        public IUnitOfWork Db { get; set; }
```
Comment (iuriikhrystiuk) [inline EventsExpress.Core/Services/RoleService.cs]: Also make it a private field.

File: EventsExpress.Db/Repo/UnitOfWork.cs [hunk 7]
```diff
@@ -1,17 +1,13 @@
...
-using System.Collections.Generic;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace EventsExpress.Db.Repo
 {
     public class UnitOfWork : IUnitOfWork
     {
-        private AppDbContext Database;
+        private readonly AppDbContext database;
```
Comment (iuriikhrystiuk) [inline EventsExpress.Db/Repo/UnitOfWork.cs:10]: Start field with underscore.

**Review by iuriikhrystiuk — CHANGES_REQUESTED** [review]

File: EventsExpress.Db/Repo/UnitOfWork.cs (hunk 7 above)
Comment (ArdeaSt) [reply EventsExpress.Db/Repo/UnitOfWork.cs:10]: Did it

File: EventsExpress.Core/Services/RoleService.cs (hunk 6 above)
Comment (ArdeaSt) [reply EventsExpress.Core/Services/RoleService.cs]: remaked

File: EventsExpress.Core/Services/Custom.ruleset (hunk 5 above)
Comment (ArdeaSt) [reply EventsExpress.Core/Services/Custom.ruleset]: removed

File: EventsExpress.Core/Services/CommentService.cs (hunk 4 above)
Comment (ArdeaSt) [reply EventsExpress.Core/Services/CommentService.cs:24]: In this case, we can use private field.

File: EventsExpress.Core/Services/CityService.cs (hunk 3 above)
Comment (ArdeaSt) [reply EventsExpress.Core/Services/CityService.cs]: did it

File: EventsExpress.Core/Services/CategoryService.cs (hunk 2 above)
Comment (ArdeaSt) [reply EventsExpress.Core/Services/CategoryService.cs]: changed name of field

File: EventsExpress.Core/IServices/ICountryService.cs (hunk 1 above)
Comment (ArdeaSt) [reply EventsExpress.Core/IServices/ICountryService.cs]: removed

**Review by iuriikhrystiuk — APPROVED** [review]

**Pull request merged**